### DISCUSSION TITLE
CSS Battle #13 – Totally Triangle

### DISCUSSION THREAD

**Discussion by meg-gutshall** [Virtual-Coffee/virtualcoffee.io#910] [CSS Battles]
### Link to battle:

[Let's battle! ⚔️](https://cssbattle.dev/play/13)

### Copy the code block below to format your comment on the discussion thread:

````md
<details>
<summary>Code Source  – <strong>score {character count}</strong></summary>
<!-- have to be followed by an empty line! -->

```html
your cssbattles code here!
```

</details>
````

### What others will see:

This will result in a nice hidden bit like so:

<details>
<summary>Code Source  – <strong>score {character count}</strong></summary>

```html
your cssbattles code here!
```
</details>

**Comment by meg-gutshall** [comment]
A lot of **code whittling** today! It was fun @rhdeck and @radturkin! @radturkin, make sure you post your awesome answer!

<details>
<summary>Inline styling with <code>html</code> and <code>body</code> – <strong>760.21 {96}</strong></summary>


```html
<html style=background:#0B2429><body bgcolor=#F3AC3C style=height:3in;rotate:135deg;margin:-407>
```
</details>

<details>
<summary>Inline styling <code>body</code> and <code>p</code> – <strong>774.56 {87}</strong></summary>


```html
<body bgcolor=#0B2429><p style=height:3in;rotate:135deg;background:#F3AC3C;margin:-415>
```
</details>

<details>
<summary>Then of course, linear gradient – <strong>811.22 {67}</strong></summary>


```html
<style>*>*{background:linear-gradient(135deg,#F3AC3C 25vw,#0B2429 0
```
</details>

**Comment by piotrszymaniec** [comment]
<details>
<summary>Clip path  – <strong>687.89 {159}</strong></summary>


```html
<div>
<style>
* {
  background:#0B2429;
  margin:0;
}
div {
  width: 141px;
  height: 141px;
  background: #F3AC3C;
  clip-path: polygon(0 0, 100% 0, 0 100%)
}
```
</details>

<details>
<summary>Clip path - minified  – <strong>715.87 {130}</strong></summary>


```html
<div><style> *{background:#0B2429;margin:0}div{width:141px;height:141px;background:#F3AC3C;clip-path:polygon(0 0, 100% 0, 0 100%)}
```
</details>

**Comment by aishwarya-mali** [comment]
<details>
<summary>Using borders  – <strong>660.03 {199}</strong></summary>


```html
<p>
<style>
  *{
    background: #0B2429;
    margin: 0;
  }
  p{
    width: 0; 
    height: 0; 
    border-style: solid;
    border-width: 140px 140px 0 0;
    border-color: #F3AC3C transparent;
  }
```

</details>

**Comment by ClJarvis** [comment]
<details>
<summary>Code Source  – <strong>100% 620.25 {313}</strong></summary>


```html
<div></div>
<style>
  body{
    background: #0B2429;
  }
  div {
    width: 1px;
    height: 1px;
    border-bottom: 70px solid transparent;
    border-right: 70px solid transparent;
    border-left: 70px solid #F3AC3c;
    border-top: 70px solid #F3AC3C;
    margin-top: -8px;
    margin-left: -8px;
  }
</style>
```

</details>

**Reply by ClJarvis** [reply]
I tired margin 0 and it didn't work. Must have put it on the div and not body. just tried on div and cut character count down. 627.74 {280}
